src/index.ts

```diff
@@ -14,9 +14,9 @@
  *  limitations under the License.
  */
 
-if ('serviceWorker' in navigator) {
+if ('serviceWorker' in navigator && process.env.NODE_ENV !== 'development') {
```

Comment:
ok for now, but some features require service workers (web share target) even in dev mode. It would be better to still register the service worker but disable caching.

We can also do a [skipWaiting](https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerGlobalScope/skipWaiting) in dev mode.

Reply:
Yeah, I agree that we should figure this out. Right now there's a full page warning that shows up if we weren't to disable the service worker.